tests/test_date_pyparser.py

```
@@ -3,997 +3,958 @@
+    p = year_strict
+
+    # Success
+    assert p.parse_string('1000')[0][0] == 'YEAR'
+    assert p.parse_string('1000')[0][1] == 1000
+    assert p.parse_string('2000')[0][1] == 2000
+    assert p.parse_string('2000a ')[0][1] == 2000
+    assert p.parse_string('20000 ')[0][1] == 2000
+    assert p.parse_string('17760704')[0][1] == 1776
+
+    # Failure
+    with pytest.raises(ParseException): p.parse_string('00')
```

Ruff (0.15.6)

[error] Multiple statements on one line (colon)

(E701)